pkg/catalog/manager/manager.go

```diff
@@ -69,6 +72,7 @@ func New(management managementv3.Interface, project projectv3.Interface) *Manage
 		ClusterCatalogLister:  management.ClusterCatalogs("").Controller().Lister(),
 		appRevisionClient:     project.AppRevisions(""),
 		bundledMode:           bundledMode,
+		ConfigMapLister:       core.ConfigMaps("cattle-system"),
```

Comment:
Same above as this is a client not a clister. Make sure you have client for CUD and lister for R as reading from cache is fast and less load on api server and etcd.